test/backend/test.py

```diff
@@ -125,6 +125,9 @@
     "test_and_bcast4v4d_cpu": (test_static_dynamic,),
 
     # Argmax
+    "test_argmax_no_keepdims_example_cpu": (test_static_dynamic,),
+    "test_argmax_keepdims_example_cpu": (test_static_dynamic,),
+    "test_argmax_default_axis_example_cpu": (test_static_dynamic,),
```

Comment:
You did add the backend test. It seems that the Krnl dialect fixed the issue I mentioned above.

Reply:
So I need not use affine expression instead?

Reply:
It seems that you do not need to.